app/src/main/AndroidManifest.xml

```diff
@@ -34,7 +33,7 @@
         <activity
             android:name=".ui.MainActivity"
             android:exported="true"
-            android:theme="@style/Theme.AndroidSwissTransfer">
+            android:theme="@style/Theme.AppStarting">
```

Comment:
I'm not sure we want to call the theme of the main activity "AppStarting" no? Maybe it made sens when it was a separated activity on other apps or something but here I have doubts. What do you think?

Reply:
This is the SplashScreen theme only, it's replaced by the `Theme.AndroidSwissTransfer` theme when the SplashScreen is done.

Reply:
_(With `postSplashScreenTheme` in `themes.xml`.)_